contributors/devel/architectural-roadmap.md

```diff
@@ -275,13 +275,13 @@ Kubernetes cannot function without this basic API machinery and semantics, inclu
   /portforward APIs.
 
 * TBD: The
-  [CertificateSigningRequest](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/kubelet-tls-bootstrap.md)
+  [CertificateSigningRequest](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/kubelet-tls-bootstrap.md)
   API, to enable credential generation, in particular to mint Kubelet
   credentials
 
 Ideally, this nuclear API server would only support the minimum
 required APIs, and additional functionality would be added via
-[aggregation](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/aggregated-api-servers.md),
+[aggregation](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md),
```

Comment:
same repo. This can be:
`[aggregation](/contributors/design-proposals/api-machinery/aggregated-api-servers.md)`